schema/shapes/path.json

```diff
@@ -0,0 +1,29 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "title": "Path",
+    "description": "Animatable Bezier curve",
+    "allOf": [
+        {
+            "$ref": "#/$defs/shapes/shape"
+        },
+        {
+            "type": "object",
+            "properties": {
+                "ty": {
+                    "title": "Shape Type",
+                    "type": "string",
+                    "const": "sh"
+                },
+                "ks": {
+                    "title": "Shape",
```

Comment:
should it be named differently? A bit of confusion with the `shape.json`

Reply:
the name of the property?

Reply:
yes

Reply:
IMO a property and an object having the same title is fine